projects/components/src/table/table.component.ts

```diff
@@ -626,18 +673,30 @@ export class TableComponent
     return !!this.data && !!this.columnConfigs && (this.pageable ? !!this.paginator : true);
   }
 
-  public showColumnResizeDivider = (columns: TableColumnConfig[], index: number): boolean => {
-    if (index === 0) {
+  public showColumnDivider = (columns: TableColumnConfig[], index: number): boolean => {
+    if (index === columns.length - 1) {
       return false;
     }
 
-    if (this.isStateColumn(columns[index - 1])) {
+    if (this.isStateColumn(columns[index])) {
       return false;
     }
 
     return true;
   };
 
+  public showColumnResizeHandler = (columns: TableColumnConfig[], index: number): boolean => {
+    if (!(this.resizable ?? true)) {
```

Comment:
This is some fun boolean logic. Seems a roundabout way of doing these checks. Is this because `resizable` can be undefined? 

I feel like a lot of the logic in `showColumnDivider` and  `showColumnResizeHandler` could be cleaned up a bit, but maybe not.

Reply:
I added functions so that we don't have to make some changes in the template and now two different methods are needed for showing the divider and resize handler as I decoupled them with each other.
It is because of resizable as well the fact that resizing can be done on the last column but divider won't be there for that and due to this reason we need one more extra check